.github/ISSUE_TEMPLATE/bug-in-sample.md

```diff
@@ -1,31 +1,32 @@
 ---
 name: Bug in Sample
 about: Report a bug in an existing sample project
-title: ''
-labels: ''
+title: '[BUG]'
+labels: 'bug'
 assignees: ''
 
 ---
-
-**Which sample is this report about?**
+## Which sample is this report about?
 - Sample name or link: 
 
-**Describe the bug**
+## Describe the bug
 <!-- Please add a clear and concise description of the problem below. -->
 
-**To Reproduce**
+## Steps to reproduce
 Steps to reproduce the behavior:
-1. 
-2. 
-3. 
+1. First step 
+2. ..
+3. ..
 
-**Expected behavior**
+## Expected behavior
 <!-- A clear and concise description of what you expected to happen. -->
 
-**Smartphone (please complete the following information):**
- - Device: 
+## Device (please complete the following information):
+ - Device (Smartphone, Tablet...):  
+ - Model:
```

Comment:
Device is e.g. Smartphone, model could be e.g OnePlus 5

Reply:
> I chose the wording because version and device could be something else (Lineage, Fire TV etc)

I see your point, still, the way it is right now is a bit confusing to me. 

IMO, in this list `Model` and `Device` are on the same level and the ordering does not necessarily imply that they are related. Same holds for `OS` and `Version`. 
We should at least be more descriptive and say `OS Version` and `Device Model` if that is what we expect the reporter to fill in.

Reply:
Can change for `OS Version` but I don't see any confusion with Device and Model.

Reply:
Yes, please adapt the `Version` field, so it is clearer what is meant :)

Reply:
https://github.com/bitmovin/bitmovin-player-android-samples/pull/73/commits/3c3e84bf420e5864e2a0c8ade36b6393d8c59eb1